File: python/66.py
class Solution:
    def plusOne(self, digits):
        """
        :type digits: List[int]
        :rtype: List[int]
        """
        ans = []
        st = ""
        for i in digits:
            st += str(i)
        s = str(int(st)+1)

        for i in range(len(s)):
            ans.append(int(s[i]))
        return anss

File: java/66.java
class Solution {
    public int[] plusOne(int[] digits) {
        int carry = 1;
        List<Integer>l = new ArrayList();
        for(int i = digits.length-1;i>=0;i--){
            int x = digits[i] + carry;
            l.add(x % 10);
            carry = x / 10;
        }
        if(carry == 1){
            l.add(carry);
        }
        int[] ans = new int[l.size()];
        for(int i = 0;i < ans.length;i++){
            ans[i] = l.get(ans.length-i-1);
        }
        return ans;
    }
}